idb/postgres/internal/writer/writer_test.go

```diff
@@ -410,6 +410,70 @@ func TestWriterTxnParticipationTableBasic(t *testing.T) {
 	}
 }
 
+func TestWriterTxnParticipationTableAppCallAddresses(t *testing.T) {
+	db, shutdownFunc := setupPostgres(t)
+	defer shutdownFunc()
+
+	block := bookkeeping.Block{
```

Comment:
I think you can use `block, err := test.MakeBlockForTxns(test.MakeGenesisBlock().BlockHeader, &stxnad)` instead of  constructing this object yourself.

Reply:
I wanted to use a round number that is not 0 or 1 for the basic test, and now that the tests are merged, I think we should keep the current construct.